.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+on: push
+jobs:
+  build:
+    runs-on: macos-latest
+    env:
+      LDFLAGS: -L/usr/local/opt/openblas/lib
+      CPPFLAGS: -I/usr/local/opt/openblas/include
+    steps:
+    - uses: actions/checkout@v2
+    - run: brew upgrade
+    - run: brew install openblas swig boost python3 tbb nlopt cminpack ceres-solver bison flex hdf5 ipopt r
```

Comment:
did you try mpfr/mpc ?

Reply:
Have a look at https://github.com/sofianehaddad/openturns/runs/2341501238?check_suite_focus=true
Both `MPFR`, `MPC` are available

Reply:
ah ok